db.go

```diff
@@ -75,12 +75,26 @@ func (self *DB) Query(sql string, vars map[string]any) (any, error) {
 	return self.send("query", sql, vars)
 }
 
+func (self *DB) SchemalessSelect(what string) ([]map[string]interface{}, error) {
+	output, err := self.Select(what)
+	if err != nil {
+		return make([]map[string]interface{}, 0), err
+	}
+
+	returnArray := make([]map[string]interface{}, 0)
```

Comment:
Since you know the # of results you should at least preallocate the slice and use the range index to assign vs appending.

Reply:
So, `output, err := self.Select(what)` returns an `any`. Therefore, in this case, we do not know the size of the array at line 84. This is why I kept it as 0. If there is a way around this, please let me know!

Reply:
is you swap this line with the next, you can use `len(rows)` as the third argument to `make([]map[string]any, 0, len(rows))` and this way you get the performance of preallocation with less code change (and somewhat more safe)

Reply:
Thank you -- updated